static/css/restyle/restyle.less

```diff
@@ -289,7 +289,7 @@ body:not(.home) .amo-header,
 .addon-details .header-bg,
 .home .header-bg {
   min-height: 130px;
-  margin-bottom: 10px;
+  margin-bottom: 0;
```

Comment:
@tofumatt 
This pull request also impacted the homepage which it shouldn't and reduced the space between the blue and white area there as well.
The margin-bottom should only be 0 on .addon-details .header-bg, but not on .home .header-bg